transcom/mymove
# INT B-18641 Display Backup Address to Office Users

## [B-18641](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A907530)

## Summary

Adds the customer's backup address to the Customer Info card on the Move Details page for office users.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Most customers populated with dev_db_e2e_populate have a backup address so you likely do not need to create any new customers.
2. Access the office application
3. Login as a Service Counselor
4. Access the Move Details page and scroll down to the Customer Info card
5. Verify the Customer Info card has a line for Backup Address and that line contains the customer's backup address (compare to the address listed on the Move History page)
6. If the customer does not have a backup address set it should display an em dash.
7. Repeat for TOO, TIO, and QAE/CSR (they all use the same component, service counselor has their own component)

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![Screenshot 2024-03-21 at 10 37 30 AM](https://github.com/transcom/mymove/assets/158500142/cb531337-f8e1-4703-b5d0-c2ee68f9b150)
